### DISCUSSION TITLE
Limiter l'apparition du mini-event de gloire aux joueurs de niveau 8 ou plus uniquement

### DISCUSSION THREAD

**Discussion by Nargacaura** [Crownicles/Crownicles#1995] [Idées acceptées / déjà implémentées]
Certains joueurs de niveau 7 ou moins peuvent tomber sur le mini-event d'infos sur la ligue et leurs récompenses alors qu'ils n'ont pas la capacité de combattre... Pourquoi pas retirer la possibilité de tomber sur ce mini-event tant qu'on n'est pas au niveau 8?

**Comment by BastLast** [comment]
Je valide, go convertir en issue
